app/src/main/res/values/settings_defaults_dont_translate.xml

```diff
@@ -4,7 +4,7 @@
     <!-- Default settings values -->
     <string name="settings_default_default_domain_text">.com</string>
     <string name="settings_default_vibrate_on_key_press_duration">0</string>
-    <string name="settings_default_pop_text_option">on_correction</string>
+    <string name="settings_default_pop_text_option">any_key</string>
```

Comment:
I do not want this to be the default: I think that the default behavior should not be a _playful_ behavior.

Reply:
"This is apart of what I said in my issue I opened, the current pop up behaviour is disturbing it is not natural. I checked and people liked it, The new behaviour feels good and natural. It has pop ups, the popup is a visual effect, the one above, not tooltip."